app/uploaders/avatar_uploader.rb

```diff
@@ -1,14 +1,18 @@
 # encoding: utf-8
 
 class AvatarUploader < CarrierWave::Uploader::Base
-  storage :sftp if Rails.env.production?
+  storage :s3 if Rails.env.production?
 
   include CarrierWave::MiniMagick
 
+  def content_type_allowlist
+    [/image\//]
+  end
+
   # Override the directory where uploaded files will be stored.
   # This is a sensible default for uploaders that are meant to be mounted:
   def store_dir
-    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+    "uploads/#{model.class.to_s.underscore}/#{model.id}"
```

Comment:
@jonodrew I don't really mind either way but any reason we removed the mounted_as from the directory?